### PR TITLE
Fix test

### PR DESCRIPTION
**Pull request by Green-Star**
Some tests timeouted when I tried to run them on my computer.
I've just increase/add some timeouts to pass them.

### CODE REVIEW

File: server/tests/api/single-pod.js [hunk 1]
```diff
@@ -187,14 +187,19 @@ describe('Test a single pod', function () {
   })
 
   it('Should have the views updated', function (done) {
-    videosUtils.getVideo(server.url, videoId, function (err, res) {
-      if (err) throw err
+    this.timeout(5000)
 
-      const video = res.body
-      expect(video.views).to.equal(1)
+    setTimeout(videosUtils.getVideo,
```
Comment (Chocobozzz) [inline server/tests/api/single-pod.js]: Are you sure this is necessary?

Comment (Green-Star) [inline server/tests/api/single-pod.js]: The failed on my laptop (video.views was equal to 0) so I add this timeout. 1000ms may be enough tough.

Comment (Chocobozzz) [inline server/tests/api/single-pod.js]: Please move the setTimeout in the previous tests (and try with 500ms):

```
 videosUtils.testVideoImage(server.url, 'video_short.webm', video.thumbnailPath, function (err, test) {
        if (err) throw err
        expect(test).to.equal(true)

        // Wait the async views increment
        setTimeout(done, 500)
})
```

So you don't need to modify the `Should have the views updated` test.

File: server/tests/api/check-params/videos.js [hunk 2]
```diff
@@ -392,6 +392,8 @@ describe('Test videos API validator', function () {
       const attach = {
         'videofile': pathUtils.join(__dirname, '..', 'fixtures', 'video_short.webm')
       }
+      this.timeout(10000)
```
Comment (Chocobozzz) [inline server/tests/api/check-params/videos.js]: Move this to the top

File: server/tests/api/friends-basic.js [hunk 3]
```diff
@@ -143,10 +143,13 @@ describe('Test basic friends', function () {
 
   it('Should not be allowed to make friend again', function (done) {
     const server = servers[1]
+    this.timeout(10000)
```
Comment (Chocobozzz) [inline server/tests/api/friends-basic.js]: Move this to the top + \n

**Pull request merged**